### DISCUSSION TITLE
Help with Windows build of CSP

### DISCUSSION THREAD

**Discussion by AdamGlustein** [Point72/csp#372] [General]
I am looking for some help in getting the CSP Windows build working for local development. I have installed Visual Studio 2022 with msvc version 19.41.34123. 

I verified the installation worked and added msvc to my PATH. I am trying to build from a Developer Powershell inside VS. When I run `cl`, I see that the compiler is in the PATH and recognized by the shell.

```
(csp) PS C:\Users\adamg\github\csp> cl
Microsoft (R) C/C++ Optimizing Compiler Version 19.41.34123 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
```

However, when I try to build using `make build-conda` (or `make build` too), I get the following error:

```
--------------------------------------------------------------------------------
-- Trying 'Visual Studio 15 2017 x64 v141' generator
--------------------------------
---------------------------
----------------------
-----------------
------------
-------
--
Not searching for unused variables given on the command line.
CMake Error at CMakeLists.txt:2 (PROJECT):
  Generator

    Visual Studio 15 2017

  could not find any instance of Visual Studio.



-- Configuring incomplete, errors occurred!
```

I am wondering why the build process is looking for Visual Studio 2017? Is there a configuration I am missing to get it to the point to the right `msvc` compiler?

To set up my dev environment, I followed the steps here https://github.com/Point72/csp/wiki/Build-CSP-from-Source#building-with-conda-on-linux. Are there extra steps needed for Windows?

Thanks
Adam

**Comment by jasonbourne13** [comment]
Hi Adam. Have you tried to feed this to chatgpt? It's very effective in solving such issues.
